wikitools/identifier_parser.py

```diff
@@ -0,0 +1,56 @@
+import typing
+
+from wikitools import link_parser
+
+ID_PREFIXES = {'#', 'id='}
+
+
+def extract_identifier(s: str) -> typing.Set[str]:
+    """
+    Attempt to extract a single object identifier from a line. Following are recognised:
+        - Implicit identifiers, which are generated by headings;
+        - Explicit identifiers: {#i-den-ti-fi-er} or {id=i-den-ti-fi-er}
+            (placed in the beginning or the end of the line: https://github.com/ppy/osu-web/issues/8057)
```

Comment:
look what clayton does with the blockquote in https://github.com/ppy/osu-wiki/issues/5440

Reply:
but that's on its own line

i meant something like `{#id} paragraph text`

Reply:
I suppose that is technically possible (as it must insert something like `<a id='id'/>`) but no we don't have that currently (and I hope we will not)

Reply:
let's remove this line either way, because it doesn't really add any useful information (and tags would also be able to be put in the middle of a paragraph anyway, i presume)

```suggestion
```